examples/joystick/Joystick.cpp

```diff
@@ -16,145 +16,191 @@ struct JoystickObject
 {
     sf::Text label;
     sf::Text value;
+
+    void draw(sf::RenderWindow& window, const sf::RenderStates& states = sf::RenderStates::Default) const
+    {
+        window.draw(label, states);
+        window.draw(value, states);
+    }
 };
 
 using Texts = std::unordered_map<std::string, JoystickObject>;
-Texts              texts;
-std::ostringstream sstr;
-float              threshold = 0.1f;
+
+auto getStrm()
```

Comment:
We prefer identifiers that use non-abbreviated works. Could you use a more verbose function name?